### PR TITLE
fix(ui): fix queue popup behaviour

### PR DESCRIPTION
**Pull request by charjac**
=== UI ===
 - move the QueuePopup component to the ui package.
 - fix positioning using getBoundaryClientRect.
 - handle stream change but it break in redux action.
 - don't display the popup when loading (didn't achieve to fix positioning when loading ...)

### CODE REVIEW

**Comment by charjac** [comment]
fix #595

**Comment by charjac** [comment]
I will add tests and a story

File: packages/ui/lib/components/QueuePopup/index.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+  return (
+    <Popup
+      className={styles.queue_popup}
+      trigger={<div onClick={handleTriggerClick}>{trigger}</div>}
+      open={isOpen}
+      onClose={handleClose}
+      onOpen={handleOpen}
+      hideOnScroll
+      position='right center'
```
Comment (nukeop) [inline packages/ui/lib/components/QueuePopup/index.js]: shouldn't this be left center?

**Comment by nukeop** [comment]
I can't reroll the stream with this change.

Btw, we could make the popup appear on right click, since double clicking moves the queue to the clicked track, and the popup interferes with that.

File: packages/ui/lib/components/QueuePopup/index.js (hunk 1 above)
Comment (charjac) [reply packages/ui/lib/components/QueuePopup/index.js]: when i put "left center", it works fine on story book but fail on the app :/
when "right center", its the opposite :scream: :scream: :scream:

**Comment by charjac** [comment]
Ok, will fix that.
No problem for the right click.

File: packages/ui/lib/components/QueuePopup/index.js (hunk 1 above)
Comment (nukeop) [reply packages/ui/lib/components/QueuePopup/index.js]: Lol, this is horrible. I'll need to replace these popups with something better at some point, but let's stick to these for the time being.

Comment (charjac) [reply packages/ui/lib/components/QueuePopup/index.js]: I think the problem come from the app, on the storybook the popup act as intended without the target hack.
Maybe a css conflict ?

**Comment by charjac** [comment]
I am not proud of this :/ but it works fine on the app, cant figure where does this position problem come ....
I still need to fix the stream change

**Comment by charjac** [comment]
what about preventing other popup when one is displayed ?

**Comment by nukeop** [comment]
I think this was already handled? If it's broken again, it'll be important to fix that.

**Comment by charjac** [comment]
ok will fix this

**Comment by nukeop** [comment]
Thanks, I appreciate this. After this I think we'll be ready for release.

**Comment by nukeop** [comment]
Ok, awesome. I need to fix the broken build soon.

**Pull request merged**